docs/topics/logging.txt

```diff
@@ -219,9 +219,17 @@ configuration merged with Django's defaults, hence you can decide if you want to
 add to, or replace the existing configuration.
 
 If the ``disable_existing_loggers`` key in the :setting:`LOGGING` dictConfig is
-set to ``True`` (which is the default) the default configuration is completely
-overridden. Alternatively you can redefine some or all of the loggers by
-setting ``disable_existing_loggers`` to ``False``.
+set to ``True`` (which is the default) then all loggers from the default
+configuration will be disabled. Alternatively you can redefine some or all of
+the loggers by setting ``disable_existing_loggers`` to ``False``.
+
+.. note::
+    There is an unintuitive aspect to how Python handles disabled loggers.
```

Comment:
Perhaps we can improve stdlib logging documentation and even the API too (for Python 3.5 only)?